students/AurelPerianu/Lesson6/test_mailroom_4.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import unittest
+import mailroom_4 as mailroom
+import os
+
+
+class MailroomTest(unittest.TestCase):
+
+    def test_donor_amount(self):
+        self.assertEqual(mailroom.donors['Donor A'], [3580, 34124.31, 7654])
+
+    def test_list_donors(self):
+        del mailroom.donors['x']
```

Comment:
Remember that your tests should be able to run in total isolation from each other -- you can use unittest's setUp and tearDown methods to deal with sharing test data across your individual test cases